nfcsedev.x

```diff
@@ -0,0 +1,201 @@
+/*++
```

Comment:
Also https://github.com/Microsoft/NFC-Class-Extension-Driver/blob/master/inc/NfcDTADev.h needs to be updated according to your changes

Reply:
I think I misunderstood. I will change that.